dist/gmusic.js

```diff
@@ -239,6 +245,87 @@ proto.playback = {
   }
 };
 
+// Create a queue API
+proto.queue = {
+  _fireEvent: function (el, etype) {
+    if (el.fireEvent) {
+      el.fireEvent('on' + etype);
+    } else {
+      var evObj = document.createEvent('Events');
+      evObj.initEvent(etype, true, false);
+      el.dispatchEvent(evObj);
+    }
+  },
```

Comment:
This is a utility more than specific to`queue`. Please relocate it to a function at the top of this file (e.g. `dispatchEvent(el, etype);`